README.md

```diff
@@ -342,6 +342,12 @@ NAME                                                      SIZE DAYS_AGO
   remote:ubuntu1404                                   270.4 Mb     88.1
 ```
 
+## Testing
+
+```
+sudo env PATH=$PATH rake
```

Comment:
I can run the tests being a user. It may be a problem with the installation. I'll take a look later on.

Reply:
It requires kernel read access to user which is not accepted in default configuration due to security issues.